packages/common/src/extent/index.ts

```diff
@@ -2,3 +2,35 @@ export * from "./get-geographic-org-extent";
 export * from "./extent-to-bbox";
 export * from "./get-org-extent-as-bbox";
 export * from "./create-extent";
+
+function isExtentCoordinateArray(extent: object) {
+  return (
+    Array.isArray(extent) &&
+    Array.isArray(extent[0]) &&
+    Array.isArray(extent[1])
+  );
+}
+
+function isExtentEnvelope(extent: any) {
+  return isExtentCoordinateArray(extent.coordinates);
+}
+
+function isExtentJSON(extent: any) {
+  return ["xmin", "ymin", "xmax", "ymax"].every(
+    key => typeof extent[key] === "number"
+  );
+}
+
+/**
+ * Check if the given extent is in a known format
+ * @param  {Object} extent extent in any format
+ * @return {Boolean}       indicator
+ */
+export function isValidExtent(extent: object) {
```

Comment:
Can we move this fn (and it's deps) to it's own file vs adding into index.ts?

Reply:
for this PR we're going to replace the `src/extent` folder w/ `src/extent.ts` by moving these fns and all fns in `/extent/*.ts` files into the new file.